src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
-}
+  const newRobot = {
+    Name: name,
+    Wheels: wheels,
```

Comment:
write names of keys from small letters
```suggestion
    name,
    wheels,
```